apps/address-service/index.js

```diff
@@ -73,5 +85,30 @@ module.exports = {
     ].filter(identity),
     configureExpress: (app) => {
         app.set('trust proxy', 1) // trust first proxy
+
+        app.use('/admin/', (req, res, next) => {
+            if (req.url === '/api') return next()
+            const cookies = nextCookie({ req })
+            const isSenderValid = hasValidJsonStructure(
+                {
+                    resolvedData: { sender: cookies['sender'] },
+                    fieldPath: 'sender',
+                    addFieldValidationError: () => null,
+                },
+                true,
+                1,
+                {
+                    fingerprint: {
+                        presence: true,
+                        format: /^[a-zA-Z0-9!#$%()*+-;=,:[\]/.?@^_`{|}~]{5,42}$/,
```

Comment:
we use to move static regexps to constant to skip extra compilations

Reply:
fixed